src/render/html.jl

```diff
@@ -25,12 +26,12 @@ function save(file::String, mime::MIME"text/html", doc::Metadata, config::Config
     end
 end
 
-type Entries
+type EntriesHtml
     entries::Vector{(Module, Any, Entry)}
 end
-Entries() = Entries((Module, Any, Entry)[])
+EntriesHtml() = EntriesHtml((Module, Any, Entry)[])
```

Comment:
Will need change to `(Module, Any, Entry)` due to those tuple changes. There's `Docile.tup` as a band-aid until the Compat.jl package updates.

```julia
type EntriesHtml
     entries::Vector{Docile.tup(Module, Any, Entry)}
end
EntriesHtml() = EntriesHtml(Docile.tup(Module, Any, Entry)[])
```

should work here, but do give me a shout if it's giving you problems.